app/controllers/static_pages_controller.rb

```diff
@@ -3,7 +3,7 @@ class StaticPagesController < ApplicationController
                      only: [:api, :copyright, :home, :status]
 
   fine_print_skip :general_terms_of_use, :privacy_policy,
-                  only: [:api, :copyright, :home, :status]
```

Comment:
Sorry I never saw this... fine print should not ask for a contract if you are not logged in, I think.